### ISSUE TITLE
MyGet Feed not directly visible

### ISSUE THREAD

**Issue by SbiCA** [Xabaril/AspNetCore.Diagnostics.HealthChecks#2]
Could you please note that you need to get the health check packages from my get (it wasn't to obvious at least for me :) )

**Comment by CarlosLanderas** [comment]
You are right. They are in myget until

**Comment by CarlosLanderas** [comment]
You are right. We will add it to the Readme. They are being published in myget until the final release arrives. Thanks :)
